### ISSUE TITLE
Cannot remove weights directory on apt-get remove

### ISSUE THREAD

**Issue by furushchev** [tork-a/darknet-release#7]
```bash
$ sudo apt-get remove --purge ros-indigo-darknet
ros-indigo-darknet (2016.11.27-1trusty) を削除しています ...
dpkg: 警告: ros-indigo-darknet の削除中、ディレクトリ '/opt/ros/indigo/share/darknet/weights' が空でないため削除できませんでした
```

```bash
$ cd /opt/ros/indigo/share/darknet/weights
$ ls -lFa
合計 263548
drwxr-xr-x 2 root root      4096  1月 24 21:45 ./
drwxr-xr-x 3 root root      4096  2月  3 14:36 ../
-rw-r--r-- 1 root root 269862452 11月 18 05:21 yolo.weights
```

maybe we need `chmod +w`? I'm now investigating...

**Comment by k-okada** [comment]
that was fixed in 2016.11.27-2trusty

--
◉ Kei Okada


2017-02-03 14:50 GMT+09:00 Furushchev <notifications@github.com>:

**Comment by furushchev** [comment]
@k-okada Oh, I did not notice that deb file is updated. I confirmed this bug is now fixed. Thanks!
